config/7.5.1/common.yaml

```diff
@@ -100,7 +100,7 @@ console:
                   - name: KIE_SERVER_CONTROLLER_OPENSHIFT_PREFER_KIESERVER_SERVICE
                     value: "true"
                   - name: KIE_SERVER_CONTROLLER_TEMPLATE_CACHE_TTL
-                    value: "60000"
```

Comment:
we can't change the old version configs or the operator won't handle prior version upgrades well... it'll detect differences in the configmaps and halt the install. this is a safety mechanism in case users customize/modify their configmaps to their liking.

please only make changes to the latest version configs.

Reply:
@tchughesiv reverted changed to previous releases, left the change only for 7.7 as per requested.